packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/ShareTopic.tsx

```diff
@@ -33,6 +32,8 @@ const style = {
 const ShareTopic = (props: Props) => {
   const {isDemo, isEmail, meetingId, stageId, appOrigin} = props
 
+  const [open, setOpen] = React.useState(false)
```

Comment:
+1 as this code will be used a lot, you can consider adding a little utility like 

```
export const useDialog = () => {
  const [isOpen, setIsOpen] = useState(false)

  const open = useCallback(() => setIsOpen(true), [])
  const close = useCallback(() => setIsOpen(false), [])

  return [isOpen, open, close] as const
```

you'll have stable refrences to helper method like open, close that are a little bit more readable than doing `setOpen(true`